### DISCUSSION TITLE
I need to listen for the editor's double-click event，what should I do?

### DISCUSSION THREAD

**Discussion by wangwufan** [facebook/lexical#4760] [Q&A]
registe a double-click command for lexical editor? or something else?Hope for help, thanks

**Comment by moy2010** [comment]
Are you using React to render the editor?

**Reply by wangwufan** [reply]
Yes

**Comment by wangwufan** [comment]
Yes,I'm using React



***@***.***
 
发件人： moy2010
发送时间： 2023-07-11 08:46
收件人： facebook/lexical
抄送： wangwufan; Author
主题： Re: [facebook/lexical] I need to listen for the editor's double-click event，what should I do? (Discussion #4760)
Are you using React to render the editor?

**Comment by wangwufan** [comment]
Solved.Customized MentionNode or other node type, with double-click events
